configure.ac

```diff
@@ -243,7 +243,9 @@ AC_TRY_LINK([#include <stdint.h>],
 dnl Check for gcc memory model aware built-in atomics
 dnl If supported check to see if not internal to compiler
 AC_MSG_CHECKING(compiler support for built-in memory model aware atomics)
-AC_TRY_COMPILE([#include <stdint.h>],
+LIBS_save="$LIBS"
+AC_SEARCH_LIBS([__atomic_load_8], [atomic])
+AC_TRY_LINK([#include <stdint.h>],
```

Comment:
I thought `AC_LINK_IFELSE` was the preferred macro these days...?

Reply:
Updated to `AC_LINK_IFELSE`.